.github/workflows/publish-canary.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Packages (canary)
+
+on:
+  workflow_dispatch:
+
+jobs:
+  canary-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2
+        with:
+          # pulls all commits and tags (needed for lerna / semantic release to correctly version)
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.16.3
+
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-yarn-cache-v0-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-cache-v0-
+
+      - name: yarn install
+        # skip the prepare step here, as lerna publish will run prepare before publishing
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > .npmrc
+          SKIP_PREPARE=true yarn install
+
+      - name: Configure CI Git User
+        run: |
+          git config --global user.name "github-actions"
+          git config --global user.email "github-actions@users.noreply.github.com"
+
+      - name: publish packages to npm
+        run: yarn run lerna publish --canary --force-publish --preid next.$(git rev-parse --short HEAD) --dist-tag next --yes
```

Comment:
We never used the `next` tag, and only used `alpha` or `beta` in the `server-wallet` package.